content/projects/gsoc/2020/projects/git-plugin-performance.adoc

```diff
@@ -23,4 +22,41 @@ links:
   idea: /projects/gsoc/2020/project-ideas/git-plugin-performance
 ---
 
-// TODO: add details here, remove `showGoogleDoc: true`
+The project aims to improve overall performance of git plugins by applying micro-benchmarking principles to create a comparitive study of CLI and JGit implementations of "git" in the plugins. The comparitive study can be used to codify the optimal selection of git implementation for an operation. Furthermore, it aims to fix non-trivial issues related to redundancy at hot code paths in git plugin.
+
+=== Project Deliverables
+* Integration of JMH: micro-benchmark module addition to test module of git plugins
+  ** This includes both utilities to support the creation of a performance benchamrking test environment in git-plugin and benchmark tests written for git operations critical to the plugin's end-to-end performance
```

Comment:
Spelling fix and optional split into two bullets rather and a single bullet with a conjunction ("and")
```suggestion
  ** Utilities to support the creation of a performance benchmarking test environment in git-plugin
  ** Benchmark tests written for git operations critical to the plugin's end-to-end performance
```